whois-update/src/main/java/net/ripe/db/whois/update/handler/transform/Transformer.java

```diff
@@ -7,7 +7,7 @@
 
 public interface Transformer {
 
-    RpslObject transform(final RpslObject rpslObject,
+    RpslObject transform(final RpslObject updatedObject,
```

Comment:
The *original* object is passed to `transform()` and the *updated* object is returned from it

Reply:
mmmm...here we have the updated object, not the original I mean we don't have here the error from the database. We have the body of the request

Reply:
No I mean the original (as in the unmodified by *this* method) object is passed to the method, it's updated if necessary and returned. Using `updatedObject` before it's updated is misleading.